CONTRIBUTING.md

```diff
@@ -0,0 +1,187 @@
+# Contributing to react-crisp
+
+First off, thanks for taking the time to contribute!
+
+The following as a set of guidelines for contributing to react-crisp.
+These are mostly guidelines, not rules. Use your best judgment, and fell free
+to propose changes to this document in a pull request.
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the
+[react-crisp Code of Conduct](CODE_OF_CONDUCT.md). By participating, you
```

Comment:
is there a COC?

Reply:
https://github.com/Mergifyio/react-crisp/pull/9